<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>704</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <property name="spacing">
     <number>7</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="gridGroupBox">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="horizontalSpacing">
        <number>5</number>
       </property>
       <property name="verticalSpacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="horizontalFrame">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="formGroupBox">
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <property name="horizontalSpacing">
              <number>7</number>
             </property>
             <property name="verticalSpacing">
              <number>8</number>
             </property>
             <property name="leftMargin">
              <number>7</number>
             </property>
             <property name="topMargin">
              <number>11</number>
             </property>
             <property name="rightMargin">
              <number>12</number>
             </property>
             <property name="bottomMargin">
              <number>11</number>
             </property>
             <item row="0" column="1">
              <widget class="QGroupBox" name="gridGroupBox_2">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>300</height>
                </size>
               </property>
               <layout class="QGridLayout">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>11</number>
                </property>
                <property name="rightMargin">
                 <number>11</number>
                </property>
                <property name="bottomMargin">
                 <number>12</number>
                </property>
                <property name="horizontalSpacing">
                 <number>16</number>
                </property>
                <item row="1" column="2">
                 <widget class="QGroupBox" name="gridGroupBox">
                  <layout class="QGridLayout" name="gridLayout_5">
                   <property name="leftMargin">
                    <number>3</number>
                   </property>
                   <property name="topMargin">
                    <number>3</number>
                   </property>
                   <property name="rightMargin">
                    <number>9</number>
                   </property>
                   <property name="bottomMargin">
                    <number>3</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>Число разбиений:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="lineEditSplitNum"/>
                   </item>
                   <item row="1" column="0" colspan="2">
                    <widget class="QPushButton" name="PushApproximate">
                     <property name="text">
                      <string>Аппроксимировать</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="4" column="1" colspan="2">
                 <widget class="QGroupBox" name="gridGroupBox">
                  <layout class="QGridLayout" name="gridLayout_6">
                   <property name="spacing">
                    <number>1</number>
                   </property>
                   <item row="1" column="0">
                    <widget class="QTextBrowser" name="textBrowser">
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="html">
                      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Справка:&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QGroupBox" name="FungListGroupBox">
                  <property name="title">
                   <string>Функция</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <layout class="QGridLayout" name="gridLayout">
                   <property name="leftMargin">
                    <number>1</number>
                   </property>
                   <property name="topMargin">
                    <number>1</number>
                   </property>
                   <property name="rightMargin">
                    <number>1</number>
                   </property>
                   <property name="bottomMargin">
                    <number>1</number>
                   </property>
                   <property name="spacing">
                    <number>7</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QListWidget" name="FuncList">
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="currentRow">
                      <number>0</number>
                     </property>
                     <item>
                      <property name="text">
                       <string>F(x) = sinx</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>F(x) = cosx</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>F(x) = tgx</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QGroupBox" name="gridGroupBox">
                  <property name="title">
                   <string>Строить графики</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_4">
                   <property name="leftMargin">
                    <number>11</number>
                   </property>
                   <property name="topMargin">
                    <number>1</number>
                   </property>
                   <property name="rightMargin">
                    <number>1</number>
                   </property>
                   <property name="bottomMargin">
                    <number>1</number>
                   </property>
                   <property name="horizontalSpacing">
                    <number>7</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QRadioButton" name="radioButtonFunc">
                     <property name="text">
                      <string>Функций</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QRadioButton" name="radioButtonFunc1">
                     <property name="text">
                      <string>Первых производных</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <widget class="QRadioButton" name="radioButtonFunc2">
                     <property name="text">
                      <string>Вторых производных</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QGroupBox" name="BorderGroupBox">
                  <property name="title">
                   <string>Граничные условия</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_2">
                   <property name="leftMargin">
                    <number>12</number>
                   </property>
                   <property name="topMargin">
                    <number>2</number>
                   </property>
                   <property name="rightMargin">
                    <number>14</number>
                   </property>
                   <property name="bottomMargin">
                    <number>2</number>
                   </property>
                   <property name="spacing">
                    <number>8</number>
                   </property>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>S''(1) = </string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLineEdit" name="S_1"/>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="S_0"/>
                   </item>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>S''(0) = </string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QGroupBox" name="gridGroupBox">
                  <layout class="QGridLayout" name="gridLayout_7">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>Точность:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="lineEdit"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QPushButton" name="pushButton">
                  <property name="text">
                   <string>Определить число разбиений</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QTableWidget" name="DataTable_1"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="horizontalFrame">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QCustomPlot" name="graph" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
